src/modules/admin/components/users/UserAuditPage.tsx

```diff
@@ -1,25 +1,134 @@
-import { Paper } from '@mui/material';
-import React from 'react';
-import UserAccessHistoryTable from './UserAccessHistoryTable';
+import { Box, Paper, Typography } from '@mui/material';
+// import { formatISO, subWeeks } from 'date-fns';
+import { useNavigate } from 'react-router-dom';
+
+import CommonToggle, { ToggleItem } from '@/components/elements/CommonToggle';
 import Loading from '@/components/elements/Loading';
+import {
+  ClientIcon,
+  EnrollmentIcon,
+} from '@/components/elements/SemanticIcons';
 import PageTitle from '@/components/layout/PageTitle';
 import NotFound from '@/components/pages/NotFound';
+import useDebouncedState from '@/hooks/useDebouncedState';
 import useSafeParams from '@/hooks/useSafeParams';
+import ClientAccessSummaryTable from '@/modules/admin/components/users/ClientAccessSummaryTable';
+import EnrollmentAccessSummaryTable from '@/modules/admin/components/users/EnrollmentAccessSummaryTable';
 import { useUser } from '@/modules/dataFetching/hooks/useUser';
+import CommonSearchInput from '@/modules/search/components/CommonSearchInput';
+import { AdminDashboardRoutes } from '@/routes/routes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+type EntityType = 'clients' | 'enrollments';
 
-const UserAuditPage: React.FC = () => {
+const toggleItems: ToggleItem<EntityType>[] = [
+  {
+    value: 'clients',
+    label: 'Clients',
+    Icon: ClientIcon,
+  },
+  {
+    value: 'enrollments',
+    label: 'Enrollments',
+    Icon: EnrollmentIcon,
+  },
+];
+
+interface Props {
+  entityType: EntityType;
+}
+const UserAuditPage: React.FC<Props> = ({ entityType }) => {
   const { userId } = useSafeParams() as { userId: string };
   const { user, loading } = useUser(userId);
+  const [search, setSearch, debouncedSearch] = useDebouncedState<string>('');
+  const navigate = useNavigate();
+  //const defaultStartDate = useMemo<string>(() => {
+  //  const today = new Date();
+  //  const twoWeeksAgo = subWeeks(today, 2);
+  //  return formatISO(twoWeeksAgo);
+  //}, []);
```

Comment:
Was going to set a default start date per the spec but decided not to. Someone could visit the audit page expecting to see a complete history and totally miss the default filter. That seems like it could cause confusion. Let's revisit if showing all events is a performance problem